### PR TITLE
Add a pypi badge

### LINKED ISSUE #52: Make a release and publish on pypi
- Make release on both GitHub (version 0.1.0) and Pypi. 
- The installation commands in readme and contributing docs should be updated, `pip install ... `
- The pypi badge should be added to readme.

### PR DESCRIPTION
**Pull request by BSchilperoort**
This PR adds the pypi version badge to the README.md.
The version has been bumped to 0.1.1, this will allow us to properly test the Action that pushes the new releases to pypi.

 - Also fixes the linter, which complains due to a change in `pyroma` (setup.cfg missing "python_requires" in`[options]`).
 - Updated the python versions in setup.cfg

Closes #52

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -9,7 +9,7 @@ authors:
     orcid: "https://orcid.org/0000-0000-0000-0000"
```
Comment (SarahAlidoost) [inline CITATION.cff:9]: Please add Yang's and your information to this file.

File: CITATION.cff [hunk 2]
```diff
@@ -9,7 +9,7 @@ authors:
...
 date-released: 20??-MM-DD
```
Comment (SarahAlidoost) [inline CITATION.cff:10]: Let's add the date in this PR too, e.g. 2022-11-24 or 25

File: README.md [hunk 3]
```diff
@@ -60,7 +61,7 @@ conda environment before submitting the the bash script. See
 ### On CRIB
 
 [CRIB](https://crib.utwente.nl/) is the ITC Geospatial Computing Platform. You
-can run the model using `Matlab` or `Octave`. Currently, running the 
+can run the model using `Matlab` or `Octave`. Currently, running the
 exceutable file on CRIB is not supported because MATLAB Runtime can not be
 installed there. See [Install PyStemmusScope](#install-pystemmusscope).
```
Comment (SarahAlidoost) [inline README.md:66]: There are two commands below that uses `pip install` from `main` branch. when pystemmusscope is published on pypi, we can install it by `pip install pystemmusscope`. Shall we fix the commands in this PR?

**Review by SarahAlidoost — CHANGES_REQUESTED** [review]
@BSchilperoort thanks for the changes and fixing the linter :+1: please see my comments.

File: README.md (hunk 3 above)
Comment (BSchilperoort) [reply README.md:66]: Done!

**Review by SarahAlidoost — APPROVED** [review]
@BSchilperoort many thanks.

**Comment by yijianzeng** [comment]
Thanks a lot, Sarah, Bart, Yang, this is cool!

**Pull request merged**